src/main/java/com/training/assignments/Operations.java

```diff
@@ -0,0 +1,120 @@
+package com.training.assignments;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+
+public class Operations {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String getCompanyReviews(String companyName){
+        HashMap<String,String> company=new HashMap<>();
+        company.put("Endee engineers","good company for production of portable and online instrument of gas analyzer");
+        company.put("Aaion automation","installation of GPS tracker");
+        company.put("Thermolab","good company for production of stabilizers and incubator");
+        company.put("Shreeji solution and services","provide solutions of fire alarm system");
+        company.put("Google","Best platform where u will get solution for everything");
+        logger.info("You have entered: "+companyName+":");
+        switch(companyName)
+        {
+            case "Endee engineers":
+                logger.info(company.get(companyName));
+                break;
+            case "Aaion automation":
+                logger.info(company.get(companyName));
+                break;
+            case "Thermolab":
+                logger.info(company.get(companyName));
+                break;
+            case "Shreeji solution and services":
+                logger.info(company.get(companyName));
+                break;
+            case "Google":
+                logger.info(company.get(companyName));
+                break;
+            default:
+                logger.info("All companies are good");
+        }
+        return company.get(companyName);
```

Comment:
What is the use of the switch, if you are directly returning,  return company.get(companyName);